testing/trino-testing/src/main/java/io/trino/testing/FaultTolerantExecutionConnectorTestHelper.java

```diff
@@ -47,6 +47,8 @@ public static Map<String, String> getExtraProperties()
                 .put("max-tasks-waiting-for-execution-per-query", "2")
                 .put("max-tasks-waiting-for-node-per-stage", "2")
                 .put("query.schedule-split-batch-size", "2")
+                .put("fault-tolerant-execution-runtime-adaptive-partitioning-enabled", "true")
+                .put("fault-tolerant-execution-runtime-adaptive-partitioning-target-task-size", "128kB") // to trigger some runtime adaptive partitioning
```

Comment:
I would suggest having separate test which enforces that. Could be whole `AbstractTestAggregations.java` and maybe more but with different engine configuration.

Make sure to add assertions which verify that runtime repartitining actually happens so we know that test actually provides coverage.

Reply:
> I would suggest having separate test which enforces that

Yes I can do that. Probably for both `AbstractTestAggregations.java` and `AbstractTestJoins.java`.

> Make sure to add assertions which verify that runtime repartitining actually happens so we know that test actually provides coverage.

Hmm I don't see an easy way to do this